edb/tools/test/cpython_hacks.py

```diff
@@ -0,0 +1,34 @@
+#
```

Comment:
Can we please give a descriptive name to this module -- `secret_hash.py` or something.

Reply:
maybe in a `cpython_hacks` directory <_<

Reply:
I think cpython_hacks is descriptive here as we are reaching into the interpreter internals specific to cpython and it's likely this wouldn't be the last such hack

Reply:
I disagree. The hack part is an implementation detail, I couldn't care less how a module works.

Reply:
Renamed to `cpython_state`. I'd like to avoid naming the whole module after the function.

Reply:
Sure, that's a good name!